ansible/playbooks/tasks/saptune.yaml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Ensure that saptune 3.0 or greater is installed
+  community.general.zypper:
+    name: 'saptune>=3.0'
+    state: present
+
+- name: Ensure conflicting services are stopped and disabled
+  ansible.builtin.systemd:
+    name: "{{ item }}"
+    state: stopped
+    enabled: no
+  loop:
+    - sapconf
+    - tuned
+
+- name: Ensure saptune is running and enabled
+  ansible.builtin.systemd:
+    name: saptune
+    state: started
+    enabled: yes
+
+- name: Ensure saptune_check executes correctly
+  ansible.builtin.command: saptune_check
+  changed_when: no    
+
+- name: Discover active solution
+  ansible.builtin.command: saptune solution enabled
+  register: saptune_status
+  changed_when: no
+
+- name: Set fact for active solution
+  ansible.builtin.set_fact:
+    solution_configured: "{{ (saptune_status.stdout | regex_search('(\\S+)', '\\1' ))[0] | default('NONE') }}" # Capture the first block on none whitespace
+    
+- name: Show configured solution
+  ansible.builtin.debug:
+    var: solution_configured
+
+# If this is a cluster node on Azure, we need to override to disable tcp timestamps, reuse and recycle.
+# This can be done by copying the sapnote file 2382421 from /usr/share/saptune/notes to /etc/saptune/override
+# The value can then override in the in the new file
+
+- name: Disable TCP timestamps, recycle & reuse
+  ansible.builtin.blockinfile:
+    path: /etc/saptune/override/2382421
+    create: yes
+    backup: yes
+    owner: root
+    group: root
+    mode: '0640'
+    marker: "<!-- {mark} ANSIBLE MANAGED BLOCK -->"
+    block: |
+     [sysctl]
+     net.ipv4.tcp_timestamps = 0
+     net.ipv4.tcp_tw_reuse = 0
+     net.ipv4.tcp_timestamps = 0
+  when: 
+    - cluster_node
+    - platform == 'azure'    
```

Comment:
Same question regarding `azure` as above.

Reply:
Yes, variables can be overwritten at run-time.